nupic/research/frameworks/greedy_infomax/models/ClassificationModel.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2021, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+#
+# This work was built off the Greedy InfoMax codebase from Sindy Lowe:
+# https://github.com/loeweX/Greedy_InfoMax
+# The original Greedy InfoMax paper can be found here:
+# https://arxiv.org/abs/1905.11786
+# ----------------------------------------------------------------------
+
+import torch.nn as nn
+
+
+class ClassificationModel(nn.Module):
+    def __init__(self, in_channels=256, num_classes=10, hidden_nodes=0):
+        super().__init__()
+        self.in_channels = in_channels
```

Comment:
Please add documentation for `ClassificationModel` and `MultipleClassificationModel`. Also, I think `Classifer` would be a better name for this class - thoughts?

Reply:
That works and I can change it. The reason for being called ClassificationModel (and all the other models' names) is that they were taken from the original codebase. Considering we are no longer even referencing the original codebase, I am happy to change it.